medusa/version_checker.py

```diff
@@ -46,6 +46,35 @@
 log.logger.addHandler(logging.NullHandler())
 
 
+def runs_in_docker():
```

Comment:
Why not keep this in `UpdateManager` and run it once on init?

Reply:
And then only use the global after?

Reply:
Possibly, yes.

Reply:
implemented the changes.